src/ExpressionEvaluator/Core/Source/ExpressionCompiler/CustomTypeInfo.cs

```diff
@@ -1,37 +1,196 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using Microsoft.CodeAnalysis.Collections;
+using Microsoft.VisualStudio.Debugger.Evaluation.ClrCompilation;
 using System;
 using System.Collections.ObjectModel;
-using Microsoft.VisualStudio.Debugger.Evaluation.ClrCompilation;
+using System.Text;
 
 namespace Microsoft.CodeAnalysis.ExpressionEvaluator
 {
-    /// <remarks>
-    /// We can't instantiate <see cref="DkmClrCustomTypeInfo"/> in the unit tests
-    /// (since we run against a reference assembly), so we use this type as an
-    /// intermediary.
-    /// </remarks>
-    internal struct CustomTypeInfo
+    internal static class CustomTypeInfo
     {
-        public Guid PayloadTypeId;
-        public ReadOnlyCollection<byte> Payload;
+        internal static readonly Guid PayloadTypeId = new Guid("108766CE-DF68-46EE-B761-0DCB7AC805F1");
```

Comment:
I'm assuming the above is just a brand new Guid and it should not have the value as the old `PayloadTypeId = new Guid("826D6EC1-DC4B-46AF-BE05-CD3F1A1FD4AC")`. Is that correct?